charts/atlas-operator/values.yaml

```diff
@@ -43,4 +43,26 @@ affinity: {}
 
 # By default, the operator will recreate devdb pods after migration
 # Set this to true to keep the devdb pods around.
-prewarmDevDB: true
\ No newline at end of file
+prewarmDevDB: true
+
+# env is a list of environment variables to set in the operator's deployment
+# env:
+#   normal:
+#     PREWARM_DEVDB: "true"
+#     ...
+#   secret:
+#     ATLAS_TOKEN: ""
+#     DATABASE_URL: ""
+#     ...
+env: 
+  # non sensitive variables
+  normal:
+  # sensitive variables
+  secret:
+
+# envSecretRef is a reference to a secret containing the operator's environment variables
+# used when you want to flexibly manage the operator's environment variables
+envSecretRef: ""
+# envConfigMapRef is a reference to a configmap containing the operator's environment variables
+# used when you want to flexibly manage the operator's environment variables
+envConfigMapRef: ""
```

Comment:
I prefer we only support two cases bellow, it cover almost of the use-cases
```yaml
envSecret: "my-secret"
extraEnvs:
  - name: "AWS_REGION"
    value: "us-west-2"
```

Additional, we can support set `atlasToken` directly, but create a built-in secret for store this value
```yaml
atlasToken: "aci_xxx"
```

```yaml
{{- if .Values.atlasToken }}
apiVersion: v1
kind: Secret
metadata:
  name: {{ include "atlas-operator.secret" . }}
  labels:
    app.kubernetes.io/created-by: atlas-operator
    app.kubernetes.io/part-of: atlas-operator
  {{- include "atlas-operator.labels" . | nindent 4 }}
type: Opaque
data:
  ATLAS_TOKEN: {{ .Values.atlasToken | b64enc | quote }}
{{- end}}
```

For the deployment, we can use `envFrom` and `env` to inject the secret and extraEnvs
```yaml
        envFrom:
          {{- if .Values.atlasToken }}
          - secretRef:
              name: "{{ include "atlas-operator.secret" . }}"
          {{- end }}
          {{- if .Values.envSecret }}
          - secretRef:
              name: "{{ .Values.envSecret }}"
          {{- end }}
        env:
          {{- if .Values.extraEnvs }}
          {{- toYaml .Values.extraEnvs | nindent 12 }}
          {{- end }}
```